.travis.yml

```
@@ -53,19 +53,24 @@ before_script:
 script:
   - make test-dirty test
 
+jobs:
+  include:
+    - stage: deploy
+      env:
+        - OPERATOR_SDK_VERSION=v0.8.1 KUBE_VERSION=v1.15.10 MINIKUBE_VERSION=v1.6.2 SHFMT_VERSION=v3.0.1
+      script: make travis-deploy-images
+    - stage: deploy-release
+      env:
+        - OPERATOR_SDK_VERSION=v0.8.1 KUBE_VERSION=v1.15.10 MINIKUBE_VERSION=v1.6.2 SHFMT_VERSION=v3.0.1
+      script: make travis-release
+
+stages:
+  - test
+  - name: deploy
+    if: type IN (push) AND branch = master
+  - name: deploy-release
+    if: tag =~ /^v\d+\.\d+\.\d+$/
```

1. [optional] The Travis docs are somewhat confusing to me, but looking at the ["Conditional Jobs"](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditional-jobs) section, I'm kinda thinking maybe we can put the `if` blocks already in the `jobs: include:` block above? and then not have to add the separate `stages:` block? that said, I think I'd try testing this stuff on some separate sandbox repo first anyway...
  2. Shouldn't we also carry over an `AND repo = KohlsTechnology/eunomia` condition, for both `deploy` and `deploy-release`? I would guess this was probably intended to avoid `make travis-deploy-images` and `make travis-release` being called during CI in forked repos? Actually, there [seems to exist][fork] a `fork = false` condition, maybe it'll make the intention clearer:

```suggestion
    if: tag =~ /^v\d+\.\d+\.\d+$/ AND fork = false
```

[fork]: https://docs.travis-ci.com/user/conditions-v1#examples

---

I mocked up a `.travis.yml` over here for testing ... https://github.com/seanmalloy/travis-testing. This is how I made sure it all works.

I'll add the additional fork condition and re-test it.

---

Fixed.